aasaan_jobs/kirmi.py

```diff
@@ -127,7 +127,7 @@ def request(self, url, headers=None, data=None, download=False):
                     cached_response = self.cache.get(cache_key)
 
                     if cached_response:
-                        logger.debug("returning response from cache : {cache_key}")
+                        logger.debug(f"returning response from cache : {cache_key}")
```

Comment:
@yashodhan19 What does adding the `f` do here? The reason I am asking is we had an issue with this, while trying to run the script.

Reply:
without the formatted string literal the cache_key will not be visible on the log.
we can stick to using  str.format